pom.xml

```diff
@@ -705,140 +717,6 @@
             </build>
         </profile>
 
-        <profile>
-            <id>test-coverage-nightly</id>
```

Comment:
I just confirmed that we're using the `test-coverage` profile for the nightly Sonar OS build.

Reply:
Perhaps we should rename the profile to 'Sonar OS build' to indicate what is being used for?

Reply:
@pveentjer I removed the profile as it's not used at all, it's a left over from the past.